tests/marine/test_marine.py

```diff
@@ -928,7 +928,9 @@ def test_validate_bpf_success(marine_instance: Union[Marine, MarinePool]):
 
 
 def test_validate_bpf_failure(marine_instance: Union[Marine, MarinePool]):
-    assert not marine_instance.validate_bpf("what is this bpf?")
+    res = marine_instance.validate_bpf("what is this bpf?")
+    assert not res
+    assert "syntax error" in res.errors[0]
```

Comment:
^ This is what I meant by overkill. I think this API isn't the most intuitive when we only expect 1 error message that is a string